src/demos/components/properties.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropertyJson from "./components_properties.json";
+import { Grid, Inner, Cell } from "../../../src/libs";
+
+const Head = ["Name", "Default", "Type"];
+
+function Property(Component) {
+  const bodySingle = [];
+  const bodyMultipleProps = [];
+  const PropsValue = [];
+  let count = 0;
+  Object.keys(PropertyJson).forEach((props) => {
+    if (props === Component.name) {
+      if (PropertyJson[props].length === undefined) {
+        Object.keys(PropertyJson[props]).forEach((GeneralPropertyName) => {
+          bodyMultipleProps.push([]);
+          PropertyJson[props][GeneralPropertyName].forEach(element => (
+            bodyMultipleProps[count].push(element)
+          ));
+          PropsValue.push(GeneralPropertyName);
+          count += 1;
+        });
+      } else {
+        PropertyJson[props].forEach(element =>
+          (
+            bodySingle.push(element)
+          ));
+      }
+    }
+  });
+  count = -1;
```

Comment:
heh, what?

Reply:
Yes It's because I've writter [count++] insted of [++count]. So count need to start a -1 to return 0 at the first itteration.

Reply:
mmh I dont understand, there is not a single valid reason to start an index at < 0.

Reply:
It's because I use only one variable to count so I need to reinitialise it after the first part of the code

Reply:
I still dont understand. Note that `.map()` has an optional second parameter that is the loop `index`.